web/src/features/fireWeather/components/SortableTableByDatetime.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableContainer from '@material-ui/core/TableContainer'
+import TableRow from '@material-ui/core/TableRow'
+import TableHead from '@material-ui/core/TableHead'
+import TableSortLabel from '@material-ui/core/TableSortLabel'
+import Paper from '@material-ui/core/Paper'
+import Typography from '@material-ui/core/Typography'
+
+import { getDatetimeComparator, Order } from 'utils/table'
+
+const useStyles = makeStyles({
+  display: {
+    paddingBottom: 12
+  },
+  paper: {
+    width: '100%'
+  },
+  tableContainer: {
+    maxHeight: 280
+  },
+  title: {
+    paddingBottom: 4
+  }
+})
+
+interface WeatherValue {
+  datetime: string
+  temperature?: number | null
+  relative_humidity?: number | null
+  wind_direction?: number | null
+  wind_speed?: number | null
+  precipitation?: number | null
+  total_precipitation?: number | null
```

Comment:
when is `precipitation` used and when is `total_precipitation` used?

Reply:
`precipitation` is for observation, `total_precipitation` is for forecast, and there will be another one for `delta_precipitation` for model